torch/onnx/_internal/exporter.py

```diff
@@ -76,6 +78,11 @@
 _DEFAULT_FAILED_EXPORT_SARIF_LOG_PATH = "report_dynamo_export.sarif"
 """The default path to write the SARIF log to if the export fails."""
 
+log = logging.getLogger(__name__)
```

Comment:
Why moved out from `ExportOptions`?

Doesn't this invalidate the premises that `Exporter` owns everything that the export needs and it is thread safe (although we don't care about thread safety lol)

One consequence of this is that `import torch` now pays the compute time to execute `loggin.getLogger` which is not strictly necessary during import time

Reply:
This is the more idiomatic way of how logger is used. Our way of owning logger instance is actually listed as [pattern to avoid](https://docs.python.org/3/howto/logging-cookbook.html#using-loggers-as-attributes-in-a-class-or-passing-them-as-parameters). 

Meanwhile, the more important reason is we are setting static logger name as the one registered with PT2 logging infra. It no longer makes sense to allow users creating and passing in their own logger.